integration/local-gadget/helpers_test.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"os/exec"
+	"time"
+
+	"google.golang.org/grpc"
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+var ErrNoCommandProvided = errors.New("no command provided")
+
+func run(commands ...string) ([]byte, error) {
+	if len(commands) == 0 {
+		return nil, ErrNoCommandProvided
+	}
+
+	cmd := exec.Command(commands[0], commands[1:]...)
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout, cmd.Stderr = &stdout, &stderr
+	if err := cmd.Run(); err != nil {
+		return nil, fmt.Errorf("running command=%v: %w", commands, err)
+	}
+	if stderr.Len() > 0 {
+		return nil, fmt.Errorf("error running command: %v", commands)
+	}
+
+	return stdout.Bytes(), nil
+}
+
+func runCRIContainer(ctx context.Context, runtime, prefix, image string, command []string) error {
```

Comment:
I haven't used it anywhere yet but since docker also supports CRI so having this generic to way to create/remove test container should work fine for us.